### PR TITLE
Extract settings import UI to :feature:settings:import

### PR DESCRIPTION
**Pull request by cketti**
Note to self: After this has been merged, create Weblate components for
- [ ] `:app:ui:base`
- [ ] `:feature:settings:import`

### CODE REVIEW

File: app/ui/legacy/src/main/java/com/fsck/k9/activity/MessageCompose.java [hunk 1]
```diff
@@ -1155,7 +1155,7 @@ public void onClick(DialogInterface dialog, int whichButton) {
                 return new AlertDialog.Builder(this)
                         .setTitle(R.string.confirm_discard_draft_message_title)
                         .setMessage(R.string.confirm_discard_draft_message)
-                        .setPositiveButton(R.string.cancel_action, new DialogInterface.OnClickListener() {
+                        .setPositiveButton(com.fsck.k9.ui.base.R.string.cancel_action, new DialogInterface.OnClickListener() {
```
Comment (wmontwe) [inline app/ui/legacy/src/main/java/com/fsck/k9/activity/MessageCompose.java:1158]: Import as `import com.fsck.k9.ui.base.R as BaseR`

File: app/ui/legacy/src/main/java/com/fsck/k9/ui/settings/account/OpenPgpAppSelectDialog.java [hunk 2]
```diff
@@ -237,7 +237,7 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
             builder.setView(LayoutInflater.from(getActivity()).inflate(
                     R.layout.dialog_openkeychain_info, null, false));
 
-            builder.setNegativeButton(R.string.cancel_action, new OnClickListener() {
+            builder.setNegativeButton(com.fsck.k9.ui.base.R.string.cancel_action, new OnClickListener() {
```
Comment (wmontwe) [inline app/ui/legacy/src/main/java/com/fsck/k9/ui/settings/account/OpenPgpAppSelectDialog.java:240]: Import as `import com.fsck.k9.ui.base.R as BaseR`

**Review by wmontwe — APPROVED** [review]
LGTM

File: app/ui/legacy/src/main/java/com/fsck/k9/activity/MessageCompose.java (hunk 1 above)
Comment (cketti) [reply app/ui/legacy/src/main/java/com/fsck/k9/activity/MessageCompose.java:1158]: Unfortunately, that's not possible because it's a `.java` file.

**Pull request merged**